### PR TITLE
Added note about 403 error

### PR DESCRIPTION
**Pull request by mooreds**
Internal: https://inversoft.slack.com/archives/C32T2QWCR/p1686260512177939

### CODE REVIEW

File: site/docs/v1/tech/apis/hosted-backend.adoc [hunk 1]
```diff
@@ -54,6 +54,13 @@ FusionAuth will set the domain on these cookies to `.example.com` where `example
 
 What this means is that FusionAuth needs to be hosted on the same domain or a subdomain or sibling domain of the application that you intend to use with these endpoints.
 
+[NOTE]
+====
+When developing against a FusionAuth cloud instance with a `.fusionauth.io` address, unless your application also lives at a `.fusionauth.io` address, you'll receive a `403` response code. This is because the hosted backend uses `SameSite` cookies, so if a request comes in that isn't begun from the same domain, it will fail.
+
+To work around this, develop locally, stand up a barebones backend with a more liberal cookie policy (https://github.com/FusionAuth/fusionauth-example-react-sdk/tree/main/server[here's an example]), or set up a link:/docs/v1/tech/installation-guide/cloud#updating-with-existing-custom-domains[custom domain name].
```
[the remark this answers is not in the document]
Comment (matt1hathcock) [reply site/docs/v1/tech/apis/hosted-backend.adoc:61]: It is solved once they register a custom domain and from what I have seen, if a customer really wants to test this they stand up a vanilla backend. We are better at speaking to it in the sales process as well. Do we want to mention the custom url bit solves that?

[the remark this answers is not in the document]
Comment (matt1hathcock) [reply site/docs/v1/tech/apis/hosted-backend.adoc:61]: Should we specify that CORS configuration changes in FusionAuth will not help with this.

[the remark this answers is not in the document]
Comment (mooreds) [reply site/docs/v1/tech/apis/hosted-backend.adoc]: Just updated based on this feedback, thanks!

**Review by lyleschemmerling — APPROVED** [review]

File: site/docs/v1/tech/apis/hosted-backend.adoc [hunk 2]
```diff
@@ -54,6 +54,15 @@ FusionAuth will set the domain on these cookies to `.example.com` where `example
 
 What this means is that FusionAuth needs to be hosted on the same domain or a subdomain or sibling domain of the application that you intend to use with these endpoints.
 
+[WARNING]
+====
+When developing against a FusionAuth cloud instance with a `.fusionauth.io` address, unless your application also lives at a `.fusionauth.io` address, you'll receive a `403` response code. This is because the hosted backend uses `SameSite` cookies, so if a request comes in that isn't begun from the same domain, it will fail.
```
Comment (robotdan) [inline site/docs/v1/tech/apis/hosted-backend.adoc:59]: The `403` is not due to our cookies, but instead a validation step we added. 

https://github.com/FusionAuth/fusionauth-app/blob/a091a509d5486ecaf4c45c37a76ea27f2d6dced0/src/main/java/io/fusionauth/app/action/app/BaseAppAction.java#L105

The intention of failing early is to let a developer know that this won't work unless they develop in the same domain.

I don't know that it is necessary to mention the `SameSite` cookie attribute here. We don't actually use `SameSite` cookies, instead we set an attribute on a cookie called `SameSite` which instructs the browser when to send the cookie to FusionAuth as it relates to cross origin request. 

The bigger message is that if you are developing across origin, you will have problems.

**Pull request merged**